hogben/utils.py

```diff
@@ -150,6 +150,27 @@ def __init__(self,
         self.models = models
         self.step = step
 
+    @classmethod
+    def from_sample(cls,
+                    sample,
+                    angle_times,
+                    inst_or_path='OFFSPEC'):
+        """
+        Get Fisher object using a sample.
+        Seperate constructor for magnetic simulation maybe? Probably depends
```

Comment:
improve docstring and add comment as a TODO

Reply:
Oops, kinda missed that one. That was actually pretty old, the comment was actually from before I implemented magnetic layers locally. 

This can be done with this same constructor, so TODO not neccesary even here. But docstring needs to be fixed for sure, that was sloppy :(